### ISSUE TITLE
Apply animation to items upon populating search via "Initial result list"

### ISSUE THREAD

**Issue by ltGuillaume** [TBog/TBLauncher#125]
**Description**

See this screen recording: https://ttm.sh/2lT.mp4
- I see a nice little animation for the quick list items when they appear
- I see the same animation for the items in the results list when they appear after clicking a filter (from the quick list)
- But I do **not** see any animation of the results list when opening the Search mode

Looks strange to me.

**Comment by TBog** [comment]
There is a fade in animation (for the background) when the result list first appears.

When the content of the result list changes the items get reordered with an animation, new items have a scale Y animation while old items just disappear without animation.

> I see a nice little animation for the quick list items when they appear

This can be toggled off. It's an animation of the whole QuickList bar, not of the items.

> I see the same animation for the items in the results list when they appear after clicking a filter (from the quick list)

What you see is a diff animation. The difference from the old content and the new content is animated. Old items disappear without animation, new items get a scale Y animation and moved items get animated to the new location by translation.

> But I do not see any animation of the results list when opening the Search mode

You don't because of the transparent background.

**Comment by ltGuillaume** [comment]
Thx for the extensive info, much appreciated 😃

> You don't because of the transparent background.

1. Aha! It's not because of the transparent background (I turned it opaque, but it was still without animation), I see now, but because of the animation speed under the Developer options: I've set the `Animator duration scale` to 0.5x since forever, which works great for all animations I've ever encountered, apart from - apparently - that one animation of the results list. When set to 1x, I can somewhat distinguish an animation on prepopulation. but it's still extremely fast.

2. While on 1x animations, the scale Y animation for results list items still look great when switching tags. The same scale Y animation on the quick list, however, is about 2-3 times slower 🤔
